### PR TITLE
docs(react): update README

### PR DESCRIPTION
**Pull request by rodet**
The [Carbon introduction on Medium](https://medium.com/carbondesign/up-running-with-carbon-react-in-less-than-5-minutes-25d43cca059e) got me started quickly, as neither the Carbon website nor the Carbon React repo helped me that quickly. I'd strongly encourage to mention it in the repo README as well as visibly on carbondesignsystem.com - or better, include the content directly into those two.

### CODE REVIEW

**Comment by joshblack** [comment]
Hey @rodet! 👋  Great suggestion, let me double-check that this is where we want to point folks and I'll also make sure to update that post with new screenshots with v10 😅  lol

**Comment by tw15egan** [comment]
Great feedback! Would be great to have this added to the README. Once Josh has gone over the article and it's up to spec this is good to go 👍 

Wondering if we should create a separate issue to get this added to the website (if it isn't already)

**Comment by rodet** [comment]
The ideal solution would be to have a shortened version of the article directly on the README and on the website Getting Started. Linking an updated post is good too 👍

**Comment by joshblack** [comment]
Pushed up some updates, curious what you think @rodet @tw15egan @aledavila 👀 

Article is up-to-date so should be good to go.

File: packages/react/README.md [hunk 1]
```diff
@@ -2,76 +2,80 @@
...
 
-Run the following command using [npm](https://www.npmjs.com/):
+## Getting started
+
+To install `carbon-components-react` in your project, you will need to run the
+following command using [npm](https://www.npmjs.com/):
 
 ```bash
-npm install -S carbon-components-react carbon-components carbon-icons
+npm install -S carbon-components carbon-components-react carbon-ison
```
Comment (tw15egan) [inline packages/react/README.md]:
```suggestion
npm install -S carbon-components carbon-components-react carbon-icon
```

File: packages/react/README.md [hunk 2]
```diff
@@ -2,76 +2,80 @@
...
 ```
 
 If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
 instead:
 
 ```bash
-yarn add carbon-components-react carbon-components carbon-icons
+yarn add carbon-components carbon-components-react carbon-ison
```
Comment (tw15egan) [inline packages/react/README.md]:
```suggestion
yarn add carbon-components carbon-components-react carbon-icon
```

File: packages/react/README.md [hunk 3]
```diff
@@ -2,76 +2,80 @@
...
+
+```js
+import { Button } from 'carbon-components-react';
+
+function MyComponent() {
+  return <Button>Example usage</Button>;
+}
+```
+
+For more information about each component, check-out our
```
Comment (tw15egan) [inline packages/react/README.md]:
```suggestion
For more information about each component, check out our
```

File: packages/react/README.md [hunk 4]
```diff
@@ -2,76 +2,80 @@
...
-4. Clicking the **ACTION LOGGER** tab at the bottom and interacting with the
-   selected component. You may see something like `onClick` which typically
-   indicates that the event handler (React prop) with the same name is called.
-   You can also expand the twistie to see the details of the event
-5. Clicking the **README** tab at the bottom. You can see some more document for
-   some components
+You can view available components that you can bring into your project over on
+our [Storybook](https://react.carbondesignsystem.com) or our
+[Website](https://www.carbondesignsystem.com/components/overview).
+
```
Comment (tw15egan) [inline packages/react/README.md]: This section is redundant to L50

**Review by tw15egan — COMMENTED** [review]
Few comments but otherwise looks great!

**Comment by rodet** [comment]
Overall it's a big improvement on the getting started experience. Thanks for the help, folks!

**Comment by joshblack** [comment]
@tw15egan updated!

**Review by tw15egan — APPROVED** [review]
Looks great! 🎉 👍

**Review by aledavila — APPROVED** [review]
looks good thank for adding this @rodet :)

**Pull request merged**